code/modules/shuttle/shuttle.dm

```diff
@@ -930,7 +930,7 @@
 	desc = "Used to control the Golem Ship."
 	circuit = /obj/item/circuitboard/shuttle/golem_ship
 	shuttleId = "freegolem"
-	possible_destinations = "freegolem_lavaland;freegolem_z5;freegolem_z6"
+	possible_destinations = "freegolem_lavaland;freegolem_space;freegolem_z6"
```

Comment:
```suggestion
	possible_destinations = "freegolem_lavaland;freegolem_space;freegolem_ussp"
```

If youre going to change this, also update `_maps\map_files\RandomRuins\SpaceRuins\ussp.dmm` to change `freegolem_z6` to `freegolem_ussp`